karapace/kafka_rest_apis/__init__.py

```diff
@@ -497,9 +497,9 @@ async def _maybe_create_async_producer(self) -> AsyncKafkaProducer:
                 except (NoBrokersAvailable, AuthenticationFailedError):
                     await producer.stop()
                     if retry:
-                        log.exception("Unable to connect to the bootstrap servers, retrying")
+                        log.warning("Unable to connect to the bootstrap servers, retrying")
                     else:
-                        log.exception("Giving up after trying to connect to the bootstrap servers")
+                        log.info("Giving up after trying to connect to the bootstrap servers")
```

Comment:
If retry is true, log level should be lower (info) than when retry is false (last try failing). Switch info and warning between each other here?

Reply:
My reasoning for `info` here is because the exception is re-raised, I actually think we shouldn't log at all here due to the re-raise. I used debug at first, but went with `info` to be less invasive to behavior.

Not a very strong opinion, and I can adjust to warning.

Reply:
Updated to use warning in both places.